data/json/items/magazine/40.json

```diff
@@ -67,7 +67,7 @@
     "id": "smg_40_mag",
     "type": "MAGAZINE",
     "name": "makeshift .40 20-round magazine",
-    "description": "A makeshift 20-round single-column steel magazine which fits Glock 22 pistol, but may fit some makeshift magazine-fed firearms as well.  Don't expect much reliability.",
+    "description": "An improvised magazine that is mostly compatible with a homemade 'Luty' submachinegun, with a simplified feed system.  It looks like it might feed 20 rounds of .40S&W.",
```

Comment:
I think we should axe the glock magazine compatibility on Luty pattern SMGs. It would require a more complicated magazine well and then more complicated glock compatible improvised magazines. Simpler mag well and magazines seems more likely given the nature of the weapon.

Reply:
Sounds good to me.